draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -41,9 +41,9 @@ informative:
 --- abstract
 
 QUIC defines a RESET_STREAM frame to abort sending on a stream. When a sender
-resets a stream, it also stops retransmitting STREAM frames for this stream in
-the of packet loss. On the receiving side, there is no guarantee that any data
-sent on that stream is delivered.
+resets a stream, it also stops retransmitting STREAM frames for this stream. On
+the receiving side, there is no guarantee that any data sent on that stream is
+delivered.
```

Comment:
I'd prefer to just add the missing "event".

Reply:
Makes sense. Done in 4a579e9.